src/api-v1.js

```diff
@@ -90,10 +91,12 @@ api.declare({
     let result;
 
     do {
+      let startTime = new Date();
```

Comment:
I think we prefer `process.hrtime` in node for measuring execution time. I wasn't sure why until [reading this just now](https://blog.tompawlak.org/measure-execution-time-nodejs-javascript). [The library's utils](https://github.com/taskcluster/taskcluster-lib-monitor/blob/master/src/utils.js#L38) have an example of how to work with this, but I think I should also:

1. add an example to the readme
2. make the library handle `hrtime`s directly?